### ISSUE TITLE
Unexposed ports cause issues

### ISSUE THREAD

**Issue by RmStorm** [pytest-docker-compose/pytest-docker-compose#32]
When a docker container exposes ports but docker-compose does not expose those ports pytest-docker-compose throws error.
For example, [this dockerFile](https://github.com/apache/couchdb-docker/blob/601191830ece69f731b7b256c02f9f804da1e6ec/2.3.1/Dockerfile) calls:
https://github.com/apache/couchdb-docker/blob/601191830ece69f731b7b256c02f9f804da1e6ec/2.3.1/Dockerfile#L139
```Dockerfile
EXPOSE 5984 4369 9100
```
Then if a service is configured like this:
```yaml
  couchdb:
    image: apache/couchdb:2.3.1
    ports:
      - 5984:5984
```
There are two ports present on the container that are unexposed, that looks like this in the container:
```python
container.ports == {'4369/tcp': None, '5984/tcp': [{'HostIp': '0.0.0.0', 'HostPort': '32872'}], '9100/tcp': None}
```

These ```None``` entries currently cause a crash.

**Comment by RmStorm** [comment]
Fixed by #33

**Comment by todofixthis** [comment]
@RmStorm Is the `dynamic-ports` branch good to be deleted?
